python/pyspark/mllib/evaluation.py

```diff
@@ -27,6 +27,8 @@ class BinaryClassificationMetrics(JavaModelWrapper):
     """
     Evaluator for binary classification.
 
+    :param scoreAndLabels: an RDD of (score, label) pairs
```

Comment:
Great! I didn't know this is the right way to make them show up in the doc. Is there an official doc for this?